projects/OT/index.js

```diff
@@ -1,15 +1,32 @@
 const swapContractV1 = '0x84c18204c30da662562b7a2c79397C9E05f942f0';
 const swapContractV2 = '0x2a98158166BE71D21Dd97e248ba670211Df9a73C';
+const futureContract = '0x57D28e11Cb2f72812A9f9DA72F2Ff868cd4B43F2';
+const osd='0x2F96F4397AdDdF9C2eeB5233a29556BE26eb9450';
```

Comment:
what is this osd?

Reply:
OSD is $pegged stable coin, backed by ETH and USDC as stable coin. OSD can be used as margin to open perps position on onchain.trade